src/frontend/src/common/styles/global.css

```diff
@@ -0,0 +1,15 @@
+/* 
+ * Styles here replace semantic-ui global styles.
+ * We should add new styles elsewhere.
+*/
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
```

Comment:
I'm not a huge fan of globally removing the underline, but I wasn't sure I would be able to track down all of the places to update the style.  This seems ok since it matches what we had before.